zulipterminal/ui_tools/boxes.py

```diff
@@ -102,6 +103,7 @@ def generic_autocomplete(self, text: str, state: Any) -> Optional[str]:
 
         for prefix, autocomplete_func in autocomplete_map.items():
             if text.startswith(prefix):
+                self.typeahead_mode = True
```

Comment:
minor: as per a comment from an earlier review cycle, this could have a `in_` prefix.

That would make it read `if self.in_typeahead_mode:` and similar elsewhere, so read more clearly.